hack/util.sh

```diff
@@ -149,7 +154,7 @@ function start_os_server {
 	${sudo} env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
 	 --master-config=${MASTER_CONFIG_DIR}/master-config.yaml \
 	 --node-config=${NODE_CONFIG_DIR}/node-config.yaml \
-	 --loglevel=4 \
+	 --loglevel=4 "${SERVER_ARGS}" \
```

Comment:
> @deads2k for example passing the --latest-images when running extended tests as part of the s2i pull request tester.

I didn't expect that to work, since the master config specifies an image format